### PR TITLE
Keep "version" in ClusterConfig examples aligned

### PR DESCRIPTION
**Pull request by guessi**
### Description

Existing sample ClusterConfigs under `examples/` having EOL/EOS version, should be updated.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [X] Manually tested
- [X] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Review by Himangini — CHANGES_REQUESTED** [review]
Nice spot ⭐
Instead of removing maybe bump the versions to more recent ones?
Also, cluster name changes are not required for examples

**Comment by guessi** [comment]
@Himangini the `default version` of `ClusterConfig` is depends on the version of `eksctl`.

I can add version to all sample configs, I'm wondering is that sounds good to you?

Maybe we should we keep `region` aligned as well? WDYT?

**Comment by guessi** [comment]
@Himangini Just update the commit to keep it simple, only `"version"` field been updated/added.

File: examples/27-oidc-provider.yaml [hunk 1]
```diff
@@ -3,8 +3,9 @@ apiVersion: eksctl.io/v1alpha5
 kind: ClusterConfig
 
 metadata:
-  name: cluster-26
+  name: cluster-27
```
Comment (guessi) [inline examples/27-oidc-provider.yaml:6]: To keep it aligned with the file name.

File: examples/30-vpc-with-ip-family.yaml [hunk 2]
```diff
@@ -4,9 +4,9 @@ apiVersion: eksctl.io/v1alpha5
 kind: ClusterConfig
 
 metadata:
-  name: cluster-2
+  name: cluster-30
```
Comment (guessi) [inline examples/30-vpc-with-ip-family.yaml:7]: To keep it aligned with the file name.

**Comment by guessi** [comment]
@Himangini I don't understand what you mean. Shouldn't we keep all the contents of examples up to date? why only update outdated version?

**Comment by guessi** [comment]
Closing as the PR kept open too long.

**Pull request closed without merging**